.github/workflows/scanner-release-vuln-update.yaml

```diff
@@ -5,6 +5,32 @@ on:
   workflow_dispatch:
 
 jobs:
+  download-nvd:
```

Comment:
nit: check-nvd-integrity

Reply:
Can you share some integrity checks you have in mind? I can follow-up on them later.